openapi.yaml

```diff
@@ -737,8 +737,8 @@ components:
       description: |
         Le processus utilisé pour tricoter le tissu, parmi la liste de choix suivants :
           - mix : Tricotage moyen (mix de métiers circulaire & rectiligne)
-          - fully-fashioned : Tricotage fully-fashioned
-          - seamless : Tricotage seamless
+          - fully-fashioned : Tricotage fully-fashioned / seamless
+          - integral : Tricotage intégral / whole garment
           - circular : Tricotage circulaire, inventaire désagrégé
           - straight : Tricotage rectiligne, inventaire désagrégé
```

Comment:
Could you add backticks around these identifiers so they're formatted as `<code>` tags?